pages/linux/unshare.md

```diff
@@ -0,0 +1,12 @@
+# unshare
+
+> Execute a command in a new namespace.
```

Comment:
Since there is more than 1 namespace, e.g the `net`, `mount`, `UTS`, etc etc.
How about:
Thoughts ?
```suggestion
> Execute a command in a new user defined namespace(s).
```